setup.py

```diff
@@ -92,7 +92,7 @@ def run(self) -> None:
             if platform == "mac":
                 wheel = "macosx_10_13_x86_64.whl"
             if platform == "linux":
-                wheel = "manylinux1_x86_64.whl"
+                wheel = "manylinux_2_27_x86_64.whl"
```

Comment:
Note for other reviewers. This means that at least glibc version 2.27 is required which is used in Ubuntu 18.04 (verified).